contributing_to_docs/doc_guidelines.adoc

```diff
@@ -128,62 +128,41 @@ ID. For example:
 
-[IMPORTANT]
+[NOTE]
 ====
-Use a verb in the task title (for example, Creating).
+For `Prerequistes`, `Next steps`, and `Additional resources` headings, use only `==` formatting in assembly files. For module files, use `.Prerequisites`, `.Next steps`, or `.Additional information` syntax to avoid TOC formatting within the module. Do not include xrefs in modules.
 ====
 
-Task modules take the following form:
-
-----
-<module_metadata>
-
-After the module file metadata, include a paragraph explaining why the user must
-perform this task. This should be 1-2 sentences maximum.
-
-If applicable, include any gotchas (things that could trip up the user or cause the task to fail).
-
-== Prerequisites
-
-* A bulleted list of prerequisites that MUST be performed before the user can complete this task.
-Skip if there isn't any related information.
-
-.Procedure
-
-. Step 1 - One command per step.
-
-. Step 2 - One command per step.
-
-. Step N
+== Writing concepts
+A _concept_ contains information to support the tasks that users want to do and
+must not include task information like commands or numbered steps. In most
+cases, create your concepts as individual modules and include them in
+appropriate assemblies.
 
-== Next steps
+Avoid using gerunds in concept titles. "About <concept>"
+is a common concept module title.
 
-You can explain any other tasks that MUST be completed after this task. You can
-skip this if there are none. Do not include xrefs. If the next steps are closely
-related to the task, you might be able to include their modules in the assembly.
+For more information about creating concept modules, see the
+link:https://redhat-documentation.github.io/modular-docs/[Red Hat modular docs reference guide] and the link:https://github.com/redhat-documentation/modular-docs/blob/master/modular-docs-manual/files/TEMPLATE_CONCEPT_concept-explanation.adoc[concept template].
 
-.Related information
+== Writing tasks
+A _task_ contains the steps that users follow to complete a process. A task, also known as a _procedure_, contains ordered steps and explicit commands. In most cases, create your tasks as individual modules and include them in appropriate assemblies.
 
-* A bulleted list of links to related information about this task. Skip if there isn't any related information.
-----
+Use a verb in the task title, such as "Creating".
 
 For more information about writing tasks, see the
-link:https://redhat-documentation.github.io/modular-docs/[Red Hat modular docs reference guide].
+link:https://redhat-documentation.github.io/modular-docs/[Red Hat modular docs reference guide] and the link:https://github.com/redhat-documentation/modular-docs/blob/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc[procedure template].
```

Comment:
Same, link to the raw version

Reply:
For the first link, link to the procedure section? https://redhat-documentation.github.io/modular-docs/#creating-procedure-modules